docs/en/create_connection/di-container-mssql.md

```diff
@@ -0,0 +1,51 @@
+# Connecting MsSQL
+
+To configure [Yii Db Mssql](https://github.com/yiisoft/db-mssql) with [di container](https://github.com/yiisoft/di) you need to create a configuration file.
```

Comment:
```suggestion
To configure [Yii DB MSSQL](https://github.com/yiisoft/db-mssql) with [DI container](https://github.com/yiisoft/di) you need to create a configuration file.
```

DI is always uppercase since it's an abbreviation. DB and MSSQL are the same.